pkg/icingadb/db.go

```diff
@@ -371,7 +371,9 @@ func (db *DB) CreateStreamed(ctx context.Context, entities <-chan contracts.Enti
 	}
 
 	sem := db.getSemaphoreForTable(utils.TableName(first))
-	return db.NamedBulkExec(ctx, db.BuildInsertStmt(first), 1<<15/len(db.BuildColumns(first)), sem, forward, nil)
+	stmt, placeholders := db.BuildInsertStmt(first)
+
+	return db.NamedBulkExec(ctx, stmt, 1<<15/placeholders, sem, forward, nil)
```

Comment:
`1<<15` could be made a named constant to better explain why this is done here (not directly related to this PR, but if you're touching the code anyways).

Reply:
Will be addressed in the code-quality PR.